src/VisualStudio/IntegrationTest/TestUtilities/InProcess/SolutionExplorer_InProc.cs

```diff
@@ -59,6 +59,23 @@ public static SolutionExplorer_InProc Create()
             };
         }
 
+        public void AddMetadataReference(string assemblyName, string projectName)
+        {
+            var referenceAssemblyFullPath = System.Reflection.Assembly.Load(assemblyName).Location;
```

Comment:
VSProject lets you give assembly qualified names to it. Why can't we use that support? This is actually sketchy because we're doing a load in a different context (unit tests) which might cause strange things to happen.